components/video/VideoPlayerView.bs

```diff
@@ -508,7 +509,7 @@ sub onVideoContentLoaded()
     ' Set subtitleTrack property if subs are natively supported by Roku
     selectedSubtitle = invalid
     for each subtitle in m.top.fullSubtitleData
-        if subtitle.Index = videoContent[0].selectedSubtitle
+        if subtitle.Index = videoContent[0].selectedSubtitle or subtitle.IsForced
```

Comment:
If the default subtitle function is returning the correct subtitle track, is this isforced check needed here?